ui-library-add-call-recording/api/transfer/BlobStorage.cs

```diff
@@ -37,6 +38,17 @@ public async Task<bool> UploadFileAsync(string blobName, Stream inStream)
                 await blobClient.UploadAsync(inStream);
                 return true;
             }
+
+            public string[] GetBlobs(string prefix)
+            {
+                var pages = this.containerClient.GetBlobs(Azure.Storage.Blobs.Models.BlobTraits.None, Azure.Storage.Blobs.Models.BlobStates.None, prefix);
```

Comment:
C# question, the docs say this is async:
![image](https://user-images.githubusercontent.com/2684369/152568660-52aa9208-e655-40c6-9ab6-fa2beb313cef.png)
[link](https://docs.microsoft.com/en-us/dotnet/api/azure.storage.blobs.blobcontainerclient.getblobs?view=azure-dotnet)

But the function has a sister function `GetBlobsAsync` that says the same thing...

Is `GetBlobs` still doing async requests but it force waits here synchronously until the async requests are finished and collected?

Reply:
nit: pages --> blobs?

Reply:
Yeah, this is just the synchronous version of the same. It can't be using `await` behind the scenes (since the `async` keyword flows transitively), but effectively it waits for the operation to complete.

Done `pages` -> `blobs`.